script.js

```diff
@@ -0,0 +1,122 @@
+}
+
+	else { document.getElementById("square1").textContent = 'X'
+       }
+
+};
+
+
+function function2(){
+
+	if (document.getElementById("square2").textContent === 'X') {
+		document.getElementById("square2").textContent = 'O';
+    document.getElementById("square2").style.color = 'white'
+}
+
+	else { document.getElementById("square2").textContent = 'X'
+       }
+
+};
+
+
+function function3(){
+	
+	if (document.getElementById("square3").textContent === 'X') {
+		document.getElementById("square3").textContent = 'O';
+    document.getElementById("square3").style.color = 'white'
+}
+
+	else { document.getElementById("square3").textContent = 'X'
+       }
+
+};
+
+
+function function4(){
+	
+	if (document.getElementById("square4").textContent === 'X') {
+		document.getElementById("square4").textContent = 'O';
+    document.getElementById("square4").style.color = 'white'
+}
+
+	else { document.getElementById("square4").textContent = 'X'
+       }
+
+};
+
+function function5(){
+	
+	if (document.getElementById("square5").textContent === 'X') {
+		document.getElementById("square5").textContent = 'O';
+    document.getElementById("square5").style.color = 'white'
+}
+
+	else { document.getElementById("square5").textContent = 'X'
+       }
+
+};
+
+function function6(){
+	
+	if (document.getElementById("square6").textContent === 'X') {
+		document.getElementById("square6").textContent = 'O';
+    document.getElementById("square6").style.color = 'white'
+}
+
+	else { document.getElementById("square6").textContent = 'X'
+       }
+
+};
+
+function function7(){
+	
+	if (document.getElementById("square7").textContent === 'X') {
+		document.getElementById("square7").textContent = 'O';
+    document.getElementById("square7").style.color = 'white'
+}
+
+	else { document.getElementById("square7").textContent = 'X'
+       }
+
+};
+
+function function8(){
+	
+	if (document.getElementById("square8").textContent === 'X') {
+		document.getElementById("square8").textContent = 'O';
+    document.getElementById("square8").style.color = 'white'
+}
+
+	else { document.getElementById("square8").textContent = 'X'
+       }
+
+};
+
+function function9(){
+	
+	if (document.getElementById("square9").textContent === 'X') {
+		document.getElementById("square9").textContent = 'O';
+    document.getElementById("square9").style.color = 'white'
+}
+
+	else { document.getElementById("square9").textContent = 'X'
+       }
+
+};
+
+function reset() {
+	document.getElementById("square1").style.color = 'blue'
+	document.getElementById("square2").style.color = 'blue'
+	document.getElementById("square3").style.color = 'blue'
+	document.getElementById("square4").style.color = 'blue'
+	document.getElementById("square5").style.color = 'blue'
+	document.getElementById("square6").style.color = 'blue'
+	document.getElementById("square7").style.color = 'blue'
+	document.getElementById("square8").style.color = 'blue'
+	document.getElementById("square9").style.color = 'blue'
+}
```

Comment:
additionally here, you have a class for each of these squares. Perhaps a for loop would be effective to dry up this portion.